django/contrib/admin/utils.py

```diff
@@ -222,6 +222,11 @@ def nested(self, format_callback=None):
         roots = []
         for root in self.edges.get(None, ()):
             roots.extend(self._nested(root, seen, format_callback))
+        # All values are not stored by key None in dict self.edges
```

Comment:
It would be nice to improve the comments around this as I'm not having an easy time understanding it. For example, why do we need to fetch the edges with key `None` first?